docs/static/logo-formidable-red.svg

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
This will be extracted to the common lander repo with https://github.com/FormidableLabs/formidable-landers/issues/12, right?